pkgs/top-level/all-packages.nix

```diff
@@ -18172,7 +18172,10 @@ with pkgs;
 
   libantlr3c = callPackage ../development/libraries/libantlr3c {};
 
-  libaom = callPackage ../development/libraries/libaom { };
+  libaom = callPackage ../development/libraries/libaom {
+    # Remove circular dependency for libavif
+    libjxl = libjxl.override { enableDocs = false; };
```

Comment:
That flag is called buildDocs.

Reply:
that's embarrassing, I thought I tested evaling it

Reply:
I guess lazy loading means it never actually checked once it started building?

anyways, fixed